bootstrap/sponge/src/main/java/org/geysermc/platform/sponge/GeyserSpongeConfiguration.java

```diff
@@ -149,6 +149,11 @@ public boolean isCacheChunks() {
         return node.getNode("cache-chunks").getBoolean(false);
     }
 
+    @Override
+    public int getCacheImages() {
+        return node.getNode("cache-skins").getInt(180);
```

Comment:
This should probably be 0 if the default config option is 0 and we're not bumping up the config version (which I don't see us needing to do).

Reply:
The default config option is 180 at the moment .

Reply:
...Not in this PR?

Reply:
https://github.com/GeyserMC/Geyser/pull/680/files#diff-68f00d4835b63f4044aa98523047c86aR92

Reply:
https://github.com/GeyserMC/Geyser/blob/aad3c2357158b4f8264c7ba4ed5676bf3a6297ef/connector/src/main/resources/config.yml#L94

Reply:
oh the config file, totally forgot about that. Well lets just set them all to 0 since it feels more logical anyway as per discord discussion. I'll do that later today and rebase.

Reply:
:+1: